src/pages/[domain]/metrics/[metricId]/index.tsx

```diff
@@ -1,9 +1,10 @@
 import { api } from "~/utils/api";
 
-export default function MetricsPage() {
+export default function MetricsPage({ metricId = "" }) {
   const { data: projects } = api.metrics.forProjects.useQuery();
```

Comment:
Let's update this to query the specific metric.

```ts
api.metrics.forProjects.useQuery({ metricId });
```
And add the input here:
https://github.com/gitcoinco/easy-retro-pgf/blob/c45100740dd756812f16eeb9f5f3d1734bab3f67/src/server/api/routers/metrics.ts#L32

And update this to [input.metricId] to only query the current metric: 
https://github.com/gitcoinco/easy-retro-pgf/blob/c45100740dd756812f16eeb9f5f3d1734bab3f67/src/server/api/routers/metrics.ts#L45

Reply:
done